charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -112,6 +113,7 @@ def new_representer(dumper, data):
     'CoreDNSDown': '.Values.kubeDns.enabled',
     'AlertmanagerDown': '.Values.alertmanager.enabled',
     'AggregatedAPIDown': 'semverCompare ">=1.18.0-0" $kubeTargetVersion',
+    'KubeProxyDown': '.Values.kubeProxy.enabled',
```

Comment:
Not sure if we need this?
The created rule looks broken if Values.kubeProxy.enabled is set to false.

Reply:
oh sorry i did not notice that is seems empty